src/App.tsx

```diff
@@ -1,24 +1,160 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { createTodo, deleteTodo, getTodos } from './api/todos';
+import { Todo, TodoData } from './types/Todo';
+import { SelectedFiltering } from './types/SelectedType';
+import { TodoList } from './TodoMain/TodoList';
+import { TodoLoadingError } from './TodoLoadingError';
+import { AddTodoForm } from './AddTodo/AddTodoForm';
 
-const USER_ID = 0;
+const USER_ID = 10397;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [isLoadingError, setIsLoadingError] = useState(false);
+  const [filteringBy, setFilteringBy] = useState(SelectedFiltering.All);
+  const [tempTodo, setTempTodo] = useState<null | TodoData>(null);
+  const [isCreatingError, setIsCreatingError] = useState(false);
+
+  const loadTodos = useCallback(async () => {
+    setIsLoadingError(false);
+
+    try {
+      const todosFromServer: Todo[] = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch (error) {
+      setIsLoadingError(true);
+    }
+  }, []);
+
+  const addTodo = useCallback(async (todoData: TodoData) => {
+    setIsCreatingError(false);
+    setTempTodo(todoData);
+
```

Comment:
I try to add todo only with one space, and I can do this....
<img width="555" alt="image" src="https://github.com/mate-academy/react_todo-app-add-and-delete/assets/113451960/eda22bbc-bc98-4eb3-88f1-46646b9c88b9">